packages/cactus-test-plugin-consortium-manual/package.json

```diff
@@ -47,7 +47,10 @@
     "webpack": "npm-run-all webpack:dev",
     "webpack:dev": "npm-run-all webpack:dev:node webpack:dev:web",
     "webpack:dev:node": "webpack --env=dev --target=node --config ../../webpack.config.js",
-    "webpack:dev:web": "webpack --env=dev --target=web --config ../../webpack.config.js"
+    "webpack:dev:web": "webpack --env=dev --target=web --config ../../webpack.config.js",
+    "test:jest": "cd ../../; yarn run test:jest:all 'packages/cactus-test-plugin-consortium-manual/src/test/typescript/(unit|integration|benchmark)/.*/*.test.ts'",
+    "test:tap": "cd ../../; yarn run test:tap:all ./packages/cactus-test-plugin-consortium-manual/src/test/typescript/integration/plugin-consortium-manual/openapi/openapi-validation.test.ts",
+    "test": "yarn run test:jest test:tap"
```

Comment:
@jenniferlianne Instead of the brittle hardcoding of relative paths with `cd` I recommend using the `--top-level` flag of the yarn run command.

Example:
```json
"test:jest": "yarn run --top-level test:jest:all 'packages/cactus-test-plugin-consortium-manual/src/test/typescript/(unit|integration|benchmark)/.*/*.test.ts'"
```

Reply:
Good idea, thanks.